frontend/package.json

```diff
@@ -5,7 +5,7 @@
   "author": "NBIS",
   "license": "GPL-3.0",
   "scripts": {
-    "start": "snowpack dev",
+    "start": "SNOWPACK_PUBLIC_API_URL=localhost:8080 snowpack dev",
```

Comment:
This looks like a regular environment variable to me and should be specified in the docker-compose file instead of hardcoded in the package.json file.

Reply:
Yes, I agree. We talked about it in the stand up and I am working on it.

Reply:
aha good I maybe missunderstood

Reply:
The api endpoint variable is now moved to the env file